frontend/src/components/permit/PermitCard.vue

```diff
@@ -158,15 +165,25 @@ p {
 }
 
 .p-card {
-  border-style: solid;
-  border-width: 1px;
-
+  border-radius: 16px;
   :deep(.p-card-content) {
     padding-bottom: 0;
   }
+
+  :deep(.p-card-body) {
+    border-style: solid;
+    border-width: 1px;
+    border-radius: 16px;
+    border-color: #05366260;
+    box-shadow: 0px 4px 4px 0px #0000001a;
+  }
+}
+
+.completed {
+  background-color: #05366210;
 }
 
 .darkgrey {
-  color: darkgrey;
+  color: #888888;
```

Comment:
UX: This doesn't look like a standard BC Grayscale color. Please consult the BC design system for prescribed colors and verify with UX.
- Ref: https://www2.gov.bc.ca/gov/content/digital/design-system/foundations/colour - Greyscale section. Quick inspection leads me to believe that the closest equivalent color would be `theme.gray70`, but you'd have to verify with UX on that.

Style: This should be something we should be pulling from our global stylesheet if possible in lieu of a specific component SCSS override if possible. May need to be revisited in the future depending on timelines.

Reply:
UX suggested this colour to improve visibility of 'Last updated' in the Permit card.